src/libs/NetworkConnection.ts

```diff
@@ -62,7 +62,7 @@ Onyx.connect({
             setOfflineStatus(true);
         } else {
             // If we are no longer forcing offline fetch the NetInfo to set isOffline appropriately
-            NetInfo.fetch().then((state) => setOfflineStatus(state.isInternetReachable === false));
+            NetInfo.fetch().then((state) => setOfflineStatus((state.isInternetReachable ?? false) === false));
```

Comment:
alternatively we could do this I assume, but either works
```suggestion
            NetInfo.fetch().then((state) => setOfflineStatus(state.isInternetReachable !== true));
```